docs/USER-DOCUMENTATION.md

```diff
@@ -133,15 +133,11 @@ run Etcher on a GNU/Linux system.
 Simulate an update alert
 ------------------------
 
-You can set the `ETCHER_FAKE_S3_LATEST_VERSION` environment variable to a valid
-semver version (greater than the current version) to trick the application into
-thinking that what you put there is the latest available version, therefore
-causing the update notification dialog to be presented at startup.
-
-Note that the value of the variable will be ignored if it doesn't match the
-release type of the current application version. For example, setting the
-variable to a production version (e.g. `ETCHER_FAKE_S3_LATEST_VERSION=2.0.0`)
-will be ignored if you're running a snapshot build, and vice-versa.
+You can set the `ETCHER_FAKE_GITHUB_LATEST_VERSION` environment variable to a
+valid semver version (greater than the current version) to trick the
+application into thinking that what you put there is the latest available
+version, therefore causing the update notification dialog to be presented at
+startup.
```

Comment:
I got rid of the fact that if the value of the variable doesn't match the current release type it gets ignored, since `electron-builder`, once its auto updates mode is enabled, allows clients to switch between production and snapshot channels at will.